content/source/docs/glossary.html.md

```diff
@@ -19,8 +19,8 @@ This page collects brief definitions of some of the technical terms used in the
 - [Argument](#argument)
 - [Attribute](#attribute)
 - [Backend](#backend)
-- [Blob Storage](#blob-storage)
 - [Block](#block)
+- [Blob Storage](#blob-storage)
```

Comment:
Hm, the original placement looks better to me?

Reply:
UGH you're right 😩 humans shouldn't be in charge of alphabetizing anyway 😤